### DISCUSSION TITLE
How do I get the parameters from a Run JavaScript File Button Action?

### DISCUSSION THREAD

**Discussion by ces3001** [mProjectsCode/obsidian-meta-bind-plugin#340] [Q&A] [answered]
According to https://www.moritzjung.dev/obsidian-meta-bind-plugin-docs/reference/buttonactions/runjavascript/ parameters can be passed to a Javascript file. But I can’t figure out how to get these parameters. None of `process.argv`, `process.env` nor `URLSearchParams`  contain them.

I’m calling it like this:
```meta-bind-button
label: test button
icon: ""
hidden: false
class: ""
tooltip: ""
id: test
style: default
actions:
  - type: js
    file: "scripts/track.js"
    args: {track:"exercise",label:"9-Fold Purification Breath"}
```

Thank you for any help!

**Comment by mProjectsCode** [comment, marked as answer]
they should be in `context.args`

**Reply by ces3001** [reply]
Awesome, thanks. Would be great to add this tidbit to the Docs.
